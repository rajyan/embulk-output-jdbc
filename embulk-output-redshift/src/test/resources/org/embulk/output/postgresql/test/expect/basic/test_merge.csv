id:long,value1:string,value2:string
12,a2,b2
14,a4,b4
16,a6,b6
