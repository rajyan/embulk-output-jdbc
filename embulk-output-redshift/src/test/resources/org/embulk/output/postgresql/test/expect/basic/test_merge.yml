table: test_merge
mode: merge
