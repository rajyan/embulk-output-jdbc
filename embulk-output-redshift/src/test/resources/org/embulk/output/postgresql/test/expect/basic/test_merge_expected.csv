11,A1,B1
12,a2,b2
13,A3,B3
14,a4,b4
16,a6,b6
